modules/http/src/main/java/org/mule/module/http/internal/listener/grizzly/GrizzlyRequestDispatcherFilter.java

```diff
@@ -49,6 +50,26 @@ public NextAction handleRead(final FilterChainContext ctx) throws IOException
         final HttpContent httpContent = ctx.getMessage();
         final HttpRequestPacket request = (HttpRequestPacket) httpContent.getHttpHeader();
 
+        // Handle Expect Continue
+        if (request.requiresAcknowledgement())
+        {
+            final HttpResponsePacket.Builder responsePacketBuilder = HttpResponsePacket.builder(request);
+            if ("100-continue".equalsIgnoreCase(request.getHeader(Header.Expect)))
```

Comment:
Use constant defined in HttpHeaders.Values for the value of the header. Maybe we should also use the Expect header name defined in HttpHeaders.Names? Use static imports.